themes/SuiteP/css/style.css

```diff
@@ -3805,37 +3871,211 @@ fieldset[disabled] .btn-primary.active {
     padding: 5px;
 }
 
-#EditView_tabs .edit .button, #EditView_tabs .id-ff-remove, #EditView_tabs .edit [type="button"] {
+#EditView_tabs .edit .button,
+#EditView_tabs .id-ff-remove,
+#EditView_tabs .edit [type="button"],
+#conditionLines [type="button"],
+#actionLines [type="button"] {
     margin-right: 4px;
     padding: 0px;
     background-color: #F08377;
     border: 1px solid #F08377;
     border-radius: 4px;
-    height: 32px;
-    width: 32px;
 }
 
 #EditView_tabs .id-ff-remove, #EditView_tabs .edit [type="radio"]  {
-    margin-top: 16px;
+    margin-top: 8px;
 }
 
 #EditView_tabs .edit [type="radio"], #EditView_tabs .edit [type="checkbox"]  {
     margin-top: 20px;
 }
 
-#EditView_tabs .id-ff-remove img, #EditView_tabs .edit [type="button"] img {
-    position: relative;
-    top: -4px;
-    left: 0;
-    height: 16px;
-    width: 16px;
+#EditView_tabs .id-ff-remove img,
+#EditView_tabs .edit [type="button"] img,
+#conditionLines [type="button"] img,
+#actionLines [type="button"] img {
+    content: url('../../../../index.php?entryPoint=getImage&themeName=SuiteP&imageName=id-ff-remove-nobg.png');
+    height: 32px;
+    width: 32px;
+}
+
+.emailaddresses > tbody:nth-child(1) > tr > td:nth-child(1) button img {
+    content: url('../../../../index.php?entryPoint=getImage&themeName=SuiteP&imageName=id-ff-add.png');
+    height: 32px;
+    width: 32px;
+}
+
+
+.emailaddresses button {
+    content: url('../../../../index.php?entryPoint=getImage&themeName=SuiteP&imageName=id-ff-add.png');
+    height: 34px;
+    width: 34px;
+    line-height: 0;
+}
+
+
+.emailaddresses > tbody:nth-child(1) > tr > td:nth-child(1) button {
+    border: 1px solid transparent;
+    border-radius: 4px;
+    background-color: #F08377;
+}
+
+
+.emailaddresses > tbody:nth-child(1) > tr > td:nth-child(1) button:hover {
+    background-color: #D66C60;
 }
 
 #EditView_tabs .edit input, #EditView_tabs .edit textarea {
     margin-top: 5px;
     margin-bottom: 5px;
 }
 
+.col-sm-6 .edit-view-field input[type="text"] {
+    width: 60% !important;
```

Comment:
The do not effect the column widths. They effect the text fields inside of the column widths.

There is a issue with making the fields display as the same widths. So that they look like the design. The important forces the text field width.

Most of the !important parameters are fixing inline / hardcoded styles that come from the fields tpls and the js files that have hard coded content in them. This is especially evident in the use of tables inside SuiteCRM. We need to move towards using using divs with css classes. As that would give us much more control over the appearance of SuiteCRM.

We could fix this by creating a tpl for each field type, moving hard coded content into tpls and then overriding them. This would allow us to add css classes to each field type. However, this is a task in itself. One thing we could do is make the relate field buttons be [inline](http://getbootstrap.com/components/#input-groups-buttons-multiple). We could also make dropown as [http://getbootstrap.com/components/#input-groups-buttons-segmented](segmented fields) for the fields have elements next to the inputs.